plugins/woocommerce-admin/client/marketing/overview-multichannel/InstalledExtensions.tsx

```diff
@@ -83,25 +82,16 @@ const InstalledExtensions = () => {
 	};
 
 	return (
-		<CollapsibleCard
-			className="woocommerce-marketing-installed-extensions-card"
-			header={ __( 'Installed extensions', 'woocommerce' ) }
-		>
+		<CollapsibleCard header={ __( 'Installed extensions', 'woocommerce' ) }>
 			{ installedPlugins.map( ( el, idx ) => {
 				return (
 					<Fragment key={ el.slug }>
-						<CardBody>
-							<ProductIcon product={ el.slug } />
-							<div className="woocommerce-marketing-installed-extensions-card__details">
-								<div className="woocommerce-marketing-installed-extensions-card__details-name">
-									{ el.name }
-								</div>
-								<div className="woocommerce-marketing-installed-extensions-card__details-description">
-									{ el.description }
-								</div>
-							</div>
-							{ getButton( el ) }
-						</CardBody>
+						<PluginCardBody
+							icon={ <ProductIcon product={ el.slug } /> }
+							name={ el.name }
+							description={ el.description }
+							button={ getButton( el ) }
+						/>
```

Comment:
💅  Maybe we can pass

`<PluginCardBody plugin={el} />` and handle all the params directly in the component?